torch/quantization/fx/standalone_module.py

```diff
@@ -0,0 +1,27 @@
+from torch.fx import GraphModule
+
+class ObservedStandaloneGraphModule(GraphModule):
```

Comment:
@jamesr66a @zdevito is this what we want?

Reply:
This looks fine to me in the absence of being able to directly customize what `symbolic_trace` returns. I've filed an issue to track that: https://github.com/pytorch/pytorch/issues/45534